### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by greenlandlab** [community/community#33967] [Other Feature Feedback, Questions, & Ideas]
⭐ ⭐ ⭐ ⭐ ⭐⭐⭐⭐⭐⭐ / ⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐
10/11
I like GitHub because...

1. It's an opensource platform 
2. I t has lot's of topics in it
3. it's easy and simple

**Comment by DylanDevelops** [comment]
It really is an amazing program! I just love when tools like git could be confusing so the program makes everything visual!
